frontend/scss/pages/_tools.scss

```diff
@@ -181,3 +187,98 @@
     }
   }
 }
+
+.navbar-course {
+  background-color: #fff;
+}
+
+.navbar-course li a {
+  color: #000;
+  font-weight: 400;
+  padding-left: 40px;
+}
+
+.hr-sect {
+	display: flex;
+	flex-basis: 100%;
```

Comment:
Ensure proper indentation, we are currently using 2 space indent for scss files.

Reply:
Done.